HowToUse.txt

```diff
@@ -1,5 +1,5 @@
 
-How to map randomize your pokemon heart gold or soul silver roms:
+How to map randomize your Pokémon HeartGold or soul SoulSilver:
```

Comment:
```suggestion
How to map randomize your Pokémon HeartGold or SoulSilver ROMs:
```
There is an extra "soul" here that should be removed.
Furthermore, adding the "ROMs" at the end makes it clear what exactly is being randomized, as the sentence is a bit awkward otherwise.
"How to randomize your HeartGold" sounds weird without a "ROMs" at the end, for example.

Reply:
Yeah, it is better being specific, thanks for the correction